yarn-project/cli/src/utils/commands.ts

```diff
@@ -360,8 +360,8 @@ export function stringify(data: Record<string, any>): string {
     (_key, val) => {
       if (typeof val === 'bigint') {
         return String(val);
-      } else if (val instanceof Fr || val instanceof AztecAddress || val instanceof EthAddress) {
-        return val.toString();
+      } else if (val && typeof val === 'object' && 'toBuffer' in val) {
+        return val.toBuffer().toString('hex');
```

Comment:
Should we add the 0x prefix?

Reply:
Ah good shout, then we can just `Fr.fromString()` directly without having to create a buffer 👍